app/views/layouts/avo/application.html.erb

```diff
@@ -47,7 +47,7 @@
         </div>
       </div>
     </div>
-    <%= turbo_frame_tag 'actions_show' %>
+    <%= turbo_frame_tag Avo::ACTIONS_TURBO_FRAME_ID %>
```

Comment:
Great! We should use this on our docs on the multiple action flux https://docs.avohq.io/3.0/actions.html#:~:text=you%20may%20want%20to%20redirect%20to%20another%20action.%20